### PR TITLE
Late semestral work is -10

### PR DESCRIPTION
**Pull request by hroncok**

### CODE REVIEW

File: tutorials/13_semestralka.md [hunk 1]
```diff
@@ -27,7 +27,9 @@ Termíny
 -------
 
  * **Téma schváleno do:** 20.12.2017 (předposlední cvičení)
- * **Odevzdáno do:** 31.1.2018 včetně
+ * **Odevzdáno do:** 31.1.2018 včetně, možnost pozdního odevzdání do 9.2.1018 s penalizací
+
+Pozdní odevzdání semestrálky do 9.2.2018 včetně s penalizací -10 bodů. Tato penalizace se nepočítá do podmínky 25 bodů za semestrálku. Je tedy možné mít 35 bodů ze smestru a 25-10 ze semestrálky a stále dostat E.
```
Comment (encukou) [inline tutorials/13_semestralka.md]: typo: `smestru`.

Comment (encukou) [inline tutorials/13_semestralka.md]: typo: `smestru`.

**Comment by hroncok** [comment]
(Squash if merging.)

**Review by MarekSuchanek — APPROVED** [review]

**Comment by hroncok** [comment]
Deployed to Edux manually.

**Comment by hroncok** [comment]
The year is wrong, fixed in 95fa023.

**Pull request merged**